examples/fetch/index.tsx

```diff
@@ -23,11 +23,55 @@ const App = () => {
     }
   };
 
+  async function streamToArrayBuffer(stream: any) {
```

Comment:
Instead of `any`, can we provide detailed type in `@lynx-js/types`?

Or if this is the same as the W3C [stream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream), use `ReadableStream` instead.